tests/integration/tests/test_sanity_checks.py

```diff
@@ -9,7 +9,7 @@
 import pytest
 from commonlib.utils import get_findings
 
-CONFIG_TIMEOUT = 120
+CONFIG_TIMEOUT = 300
```

Comment:
why do you change the timeout?

Reply:
The workflow failed multiple times to achieve some resources.
Most of the resources are being collected by a single fetcher (gcp_asset_fetcher), therefore, the entire flow takes more time than AWS CSPM.

Reply:
We can manage a separate timeout definition for GCP findings using `GCP_CONFIG_TIMEOUT`.